src/MusicStore/Components/SystemClock.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+
+namespace MusicStore.Components
+{
+    /// <summary>
+    /// Provides access to the normal system clock.
+    /// </summary>
+    public class SystemClock : ISystemClock
+    {
+        /// <inheritdoc />
+        public DateTime UtcNow
+        {
+            get
+            {
+                // The clock measures whole seconds only, to have integral expires_in results, and
```

Comment:
Eh ok perhaps it's fine to leave it (it's not wrong, it just looks weird).

However, please update the comment to avoid mention of "expires_in" because that's an auth concept that is irrelevant here.